app/src/components/GrantListWithFilter.vue

```diff
@@ -0,0 +1,102 @@
+<template>
```

Comment:
Just curious, why did you decide to use a new file here here instead of putting the logic directly in `GrantRegistryList.vue`?